testapps/php56_custom/php56_custom.yaml

```diff
@@ -0,0 +1,27 @@
+schemaVersion: "1.0.0"
+
+commandTests:
+  - name: "pwd"
+    command: ["pwd"]
+    expectedOutput: ["/app\n"]
+
+  # - name: "interactive output"
+  #   command: ["cat", "callable_output.txt"]
+  #   expectedOutput: ["The script is not interactive!"]
+
+  # - name: "php cli ini"
+  #   command: ["cat", "cli-ini-test-.txt"]
+  #   expectedOutput: ["shell_exec succeeded"]
+
+  # - name: "command output"
+  #   command: ["cat", "script_output.txt"]
+  #   expectedOutput: ["Testing Post Deploy Command"]
+
+  # - name: "file permissions"
+  #   command: ["stat", "-c", "'%a %n'", "script_output.txt"]
+  #   expectedOutput: ["777 script_output.txt"]
```

Comment:
since these will (probably) never work with structure tests given that the container needs to be started for these to pass, you can probably go ahead and delete them.